### DISCUSSION TITLE
Could not find module ... satisfying ...

### DISCUSSION THREAD

**Discussion by dawsonc623** [parcel-bundler/parcel#6228] [Q&A]
Specifically:

```
@parcel/package-manager: Could not find module "@parcel/transformer-sass" satisfying 2.0.0-beta.2.
/me/myproject/package.json:20:5
  19 |   "devDependencies": {
> 20 |     "@parcel/transformer-sass": "^2.0.0-beta.2",
>    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^ Found this conflicting local requirement.
```
Using `"parcel": "^2.0.0-beta.2"`.

The versions match, so I have not the faintest idea what it is complaining about, to be honest.

**Comment by mischnic** [comment]
That is indeed strange. 

Can you share more details?

**Reply by dawsonc623** [reply]
There is no other output besides that, even with `--log-level verbose`.

For some background (if it is relevant), this is a brand new React app using SASS and TypeScript. It is also part of a Lerna monorepo which publishes to a private NPM registry, so there is a `.npmrc` file with those details in it. I am not sure if there is any other detail that would be useful.

**Reply by mischnic** [reply]
> monorepo 

There's also https://github.com/parcel-bundler/parcel/issues/5854, but your error message is different.

**Reply by dawsonc623** [reply]
I will have to talk with my team to see if installing parcel at the root is something we can do to see if that fixes the issue. Is there anything in the works for getting more intuitive results with Parcel in monorepos?

**Reply by mischnic** [reply]
> Is there anything in the works for getting more intuitive results with Parcel in monorepos?

This current situation isn't ideal, but as I've said in https://github.com/parcel-bundler/parcel/issues/5854#issuecomment-779955980, I'm not really sure how it *should* work.

**Reply by dawsonc623** [reply]
Installing at the root yields the same result, both with and without `--no-autoinstall`. Not sure if it makes a difference, but this project is using `npm` instead of `yarn` - I have not used `npm` in recent years (I changed to `yarn` a few years ago and never looked back), but I was told that it should not make a difference.

**Reply by dawsonc623** [reply]
Documenting this here if anyone else stumbles across this. I cannot find an avenue forward with Parcel in the context of a Lerna monorepo, so I am going to try Webpack instead. I am not sure what else can be done.

**Reply by rrrnld** [reply]
I am stumbling across the same issue:
```
🚨 Build failed.
@parcel/package-manager: Could not find module "@parcel/transformer-sass" satisfying 2.0.0-beta.2.
/home/arne/dev/rbb/2021/gender-care-gap/package.json:54:5
  53 |   "devDependencies": {
> 54 |     "@parcel/transformer-sass": "^2.0.0-beta.2",
>    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^ Found this conflicting local requirement.
  55 |     "@testing-library/react": "^11.2.6",
  56 |     "@types/chai": "^4.2.17",
  ```

My `package.json` looks like this:
```
{
  "dependencies": {
    "@babel/core": "^7.13.16",
    "@fortawesome/fontawesome-svg-core": "^1.2.17",
    "@fortawesome/free-solid-svg-icons": "^5.8.1",
    "@fortawesome/react-fontawesome": "^0.1.4",
    "@react-spring/web": "^9.1.2",
    "@turf/turf": "^5.1.6",
    "chroma-js": "^2.0.3",
    "d3-array": "^2.12.1",
    "d3-dsv": "^1.2.0",
    "d3-force": "^2.1.1",
    "d3-scale": "^3.2.3",
    "d3-shape": "^2.0.0",
    "d3-time": "^2.1.1",
    "fuse.js": "^3.4.4",
    "intersection-observer": "^0.12.0",
    "leaflet": "^1.7.1",
    "leaflet.markercluster": "^1.4.1",
    "lodash": "^4.17.20",
    "polished": "^3.6.5",
    "prop-types": "^15.7.2",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-is": "^16.13.1",
    "react-leaflet": "^2.2.1",
    "react-leaflet-bing-v2": "^5.0.1",
    "react-scripts": "^3.4.4",
    "react-text-transition": "^1.3.0",
    "react-transition-group": "^4.4.1",
    "sass": "^1.29.0",
    "scrollama": "^2.2.2",
    "storybook": "^6.2.9"
  },
  "devDependencies": {
    "@parcel/transformer-sass": "^2.0.0-beta.2",
    "@testing-library/react": "^11.2.6",
    "@types/chai": "^4.2.17",
    "@types/mocha": "^8.2.2",
    "@types/react": "^17.0.4",
    "@types/react-dom": "^17.0.3",
    "chai": "^4.3.4",
    "cross-env": "^7.0.3",
    "parcel": "^2.0.0-beta.2",
    "postcss": "^8.2.13",
    "prettier": "^2.2.1",
    "typescript": "^4.2.4"
  }
}
```

*However* this only occurs with [pnpm](https://pnpm.io/), which I'm using to save previous disk space on my laptop. Switching back to `npm` let me run `parcel serve` without errors.

```
$ pnpm --version
5.5.10
$ npm --version
6.14.11
```

**Reply by glingy** [reply]
This just happened to me, exact same error, but for `"@parcel/transformer-vue": "2.0.0-rc.0"`
 instead. Deleting `node_modules` and running `yarn` again seemed to fix the problem.
